.eslintignore

```diff
@@ -27,10 +27,11 @@ packages/next-codemod/**/*.js
 packages/next-codemod/**/*.d.ts
 packages/next-env/**/*.d.ts
 packages/create-next-app/templates/**
-test/integration/eslint/**
-test/integration/script-loader/**/*
-test/development/basic/legacy-decorators/**/*
+test/integration/eslint/**/*.js
+test/integration/script-loader/**/*.js
+test/development/basic/legacy-decorators/**/*.js
```

Comment:
Adding the file extensions was needed for the unignore statement below to take effect.